soql-analyzer/src/test/scala/com/socrata/soql/SoQLAnalyzerTest.scala

```diff
@@ -567,4 +567,10 @@ SELECT visits, @x2.zx
     analysis.selection.head._1.name must equal("sum_a1_id")
   }
 
+
+  test("Merge window function with group by aggregate") {
```

Comment:
are these two selects supposed to be equivalent? it's not sure right off that they are

Reply:
Good news, they seem to be identical - at least in blist_dev

SELECT view_type, row_number() over(order by sum(id)) as rn from lenses GROUP BY view_type, display_type 

vs

    SELECT view_type, row_number() over(order by n2) as rn from (
    SELECT view_type, display_type, sum(id) as n2  from lenses GROUP BY view_type, display_type) t1

Reply:
btw, this has been this way all long.  it was broken shortly by window function.  it was fixed and this new test was added.